pkg/test/runner/runner.go

```diff
@@ -23,8 +23,8 @@ const (
 	expectedResultsFile string = "results.yaml"
 	expectedDiffFile    string = "diff.patch"
 	expectedConfigFile  string = "config.yaml"
-	CommandFn           string = "fn"
-	CommandPipeline     string = "pipeline"
+	CommandFnEval       string = "run"
+	CommandFnRender     string = "render"
```

Comment:
Shouldn't this be `CommandPipelineRender`?

Reply:
These are actually now `fn` subcommands. We no longer have `pipeline run` command.